crates/loam-cli/src/commands/build/clients.rs

```diff
@@ -333,4 +348,74 @@ export default new Client.Client({{
 
         Ok(())
     }
+
+    async fn run_init_script(
+        &self,
+        name: &str,
+        contract_id: &str,
+        init_script: &str,
+    ) -> Result<(), Error> {
+        for line in init_script.lines() {
+            let line = line.trim();
+            if line.is_empty() {
+                continue;
+            }
+
+            let mut args = vec!["--id", contract_id];
+            let mut source_account: Option<&str> = None;
+
+            let parts: Vec<&str> = line.split_whitespace().collect();
+            let mut command_parts = vec!["--"];
+
+            for part in &parts {
+                if let Some(value) = part.strip_prefix("STELLAR_ACCOUNT=") {
+                    source_account = Some(value);
+                } else if let Some(value) = part.strip_prefix("SOROBAN_ACCOUNT=") {
```

Comment:
I don't think the CLI supports `SOROBAN_ACCOUNT` anymore , and I'd rather have less to explain/support/maintain. Should we remove this?

Reply:
Yeah I think its best to just stick with what the CLI supports. I just had SOROBAN in there because it was in the docs that I was going off.